backend/src/apiserver/archive/log.go

```diff
@@ -52,6 +52,9 @@ type RunLogEntry struct {
 	Timestamp time.Time `json:"timestamp,omitempty"`
 }
 
+// Inspired on fluent-bit parsers
+// https://github.com/fluent/fluent-bit/blob/master/tests/runtime/data/kubernetes/parsers.conf
+// Apache License, Version 2.0, January 2004
```

Comment:
Would you mind copying their full Apache License inline?

Reply:
Sure, license inlined